src/checkpointTrie.ts

```diff
@@ -144,9 +144,18 @@ export class CheckpointTrie extends BaseTrie {
     return new ScratchReadStream(trie)
   }
 
-  // formats node to be saved by levelup.batch.
-  // returns either the hash that will be used key or the rawNode
+  /**
+   * Formats node to be saved by levelup.batch.
+   * @method _formatNode
+   * @private
+   * @param {TrieNode} node - the node to format
+   * @param {Boolean} topLevel - if the node is at the top level
+   * @param {BatchDBOp[]} opStack - the opStack to push the node's data
+   * @param {Boolean} remove - whether to remove the node (only used for CheckpointTrie)
+   * @returns {Buffer | (EmbeddedNode | null)[]} - the node's hash used as the key or the rawNode
+   */
   _formatNode(node: TrieNode, topLevel: boolean, opStack: BatchDBOp[], remove: boolean = false) {
+    console.log('ENTERING HERE')
```

Comment:
Here is a console.log left. Is this otherwise ready for review?

Reply:
thanks got it. yes ready for review, i'll publish, just ended up being some typedoc additions and typo fixes.